src/zkey/ep11.c

```diff
@@ -691,14 +743,14 @@ int ec_key_clr2sec_ep11(struct ep11_lib *ep11, unsigned int curve,
 
 	/* Add info to ep11 header (overlay over session ID field). This info
 	 * is later needed for reencipher. */
-	ep11hdr = (struct ep11kblob_header *)&blob;
-	ep11hdr->len = blob_len;
+	ep11hdr = (struct ep11kblob_header *)secure_key;
```

Comment:
The comment above is now wrong (overlay over session ID field

Reply:
Corrected.